dropwizard-lifecycle/src/main/java/io/dropwizard/lifecycle/setup/LifecycleEnvironment.java

```diff
@@ -61,6 +67,25 @@ public ExecutorServiceBuilder executorService(String nameFormat, ThreadFactory f
         return new ExecutorServiceBuilder(this, nameFormat, factory);
     }
 
+    public ExecutorService virtualExecutorService(String nameFormat) {
```

Comment:
Maybe we should add overloads for the `ScheduledExecutorService`s too.

Reply:
That's a good idea. I can fix that after our discussion about builders are complete because kind of depends on that. `poolSize` in the builder also would be irrelevant in this case. Potentially `RejectedExecutionHandler` too, though I haven't read up on that much.

Reply:
After further looking into this, there's no such factory method for virtual `ScheduledExecutorService` objects in the `Executors` class. We _could_ provide a single threaded dispatching `ScheduledExecutorService` that forwards to a `newVirtualThreadPerTaskExecutor()`, but I don't think that will be very convenient.

Reply:
It's already kind of possible to do it by passing Virtual